website/website/assets/js/map-en.js

```diff
@@ -147,30 +155,27 @@ function buildMap(...charts){
 		
 		// get dataset ID
 		//dataset = layer.feature.properties.featurecla; // = Admin-0 country
-		dataset = "regions" // = Admin-0 country
-		selectedDataset = dataset
+		selectedDataset = "regions" // = Admin-0 country
```

Comment:
The comment is probably not true. Features of the dataset regions are not only admin boundaries.